htdocs/langs/fr_FR/admin.lang

```diff
@@ -826,7 +826,8 @@ Permission402=Créer/modifier les avoirs
 Permission403=Valider les avoirs
 Permission404=Supprimer les avoirs
 Permission430=Utilisez la barre de débogage
-Permission511=Lire les règlements de salaires
```

Comment:
@orebd you need to create en_US lang, fr_FR will be overwited by transifex process

Reply:
Hello,
I add `Permission510=Read payments of salary` into all admin.lang
I'm right ?
I'm not sure to understand what you mean.

Reply:
only en_US lang is needed, other langs files are built from transifex

Reply:
Understood.
Thank you